src/models/core/Endpoints.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * URLs to various Answers services.
```

Comment:
nit: We may want to provide some more info now that this is public. Maybe something like: "Overrides for the URLs which are used when making requests to the Answers API "